test/built-ins/Array/prototype/find/return-abrupt-from-this-length.js

```diff
@@ -0,0 +1,37 @@
+// Copyright (C) 2015 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 22.1.3.8
+description: >
+  Return abrupt from ToLength(Get(O, "length")).
+info: >
+  22.1.3.8 Array.prototype.find ( predicate[ , thisArg ] )
+
+  1. Let O be ToObject(this value).
+  2. ReturnIfAbrupt(O).
+  3. Let len be ToLength(Get(O, "length")).
+  4. ReturnIfAbrupt(len).
+---*/
+
+var o = {};
+
+Object.defineProperty(o, 'length', {
+  get: function() {
+    throw new Test262Error();
+  },
+  configurable: true
+});
+// predicate fn is given to avoid false positives
```

Comment:
Since your expectation is for the user-defined `Test262Error`, you don't have to worry about false positives (a missing predicate would throw a `TypeError`)